### PR TITLE
:mortar_board: Topic10 Add section on queue operations

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Add a section on the abstract queue operations: `enqueue`, `dequeue`, and `first`. 

### Where
Topic10.rst

### Additional Notes
[It was decided to use the traditional naming for queue operations.](https://en.wikipedia.org/wiki/Queue_(abstract_data_type))

### CODE REVIEW

**Comment by twentylemon** [comment]
> [It was decided to use the traditional naming for queue operations.](https://en.wikipedia.org/wiki/Queue_(abstract_data_type))

For reference, java's Queue interface doesn't use the typical names, since a Queue is many more things (priority queue, concurrent/blocking, fixed size). The [javadoc](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Queue.html) spells it out quite nicely.

File: src/site/topic10.rst [hunk 1]
```diff
@@ -5,6 +5,31 @@ Topic #10 --- The Queue ADT
...
+    * **Dequeue**
+        * Remove something from the front/head
+    * **First**
+        * Look at the thing at the front/head of the queue, but do not remove it
+
+* We will also want our ``isEmpty``, ``size``, and ``toString``
+
+.. warning::
+
+    If you look at `Java's Queue Interface <https://docs.oracle.com/javase/7/docs/api/java/util/Queue.html>`_, you will
```
Comment (twentylemon) [inline src/site/topic10.rst:25]: Java 7 link. Strongly prefer [jdk11](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Queue.html).

**Review by twentylemon — COMMENTED** [review]
> > [It was decided to use the traditional naming for queue operations.](https://en.wikipedia.org/wiki/Queue_(abstract_data_type))
> 
> For reference, java's Queue interface doesn't use the typical names, since a Queue is many more things (priority queue, concurrent/blocking, fixed size). The [javadoc](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Queue.html) spells it out quite nicely.

Look at this guy, writing stuff before he reads the code.

File: src/site/topic10.rst (hunk 1 above)
Comment (jameshughes89) [reply src/site/topic10.rst:25]: Good catch.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**